Cesium3DTiles/src/Tile.cpp

```diff
@@ -104,21 +107,138 @@ bool Tile::isRenderable() const noexcept {
       return std::all_of(
           this->_rasterTiles.begin(),
           this->_rasterTiles.end(),
-          [](const RasterMappedTo3DTile& rasterTile) {
-            return rasterTile.getReadyTile() != nullptr;
+          [](const std::unique_ptr<RasterMappedTo3DTile>& rasterTile) {
+            return rasterTile != nullptr &&
+                   rasterTile->getReadyTile() != nullptr;
           });
     }
   }
 
   return false;
 }
 
+static void applyCustomMasks(
```

Comment:
Is this part of the old implementation and is going away? If not, I don't understand what purpose it serves. Which may just mean I need to look at it closer.

Reply:
Yup this part will go away!